boggs/src/main/java/com/hartwig/healthchecks/boggs/adapter/BoggsAdapter.java

```diff
@@ -27,7 +26,7 @@
 	private static final String IO_ERROR_MSG = "Got IO Exception with message: %s";
 	private static final String EMPTY_FILE_ERROR_MSG = "Got Empty File Exception with message: %s";
 
-	private Report report = JsonReport.getInstance();
+	private final Report report = JsonReport.getInstance();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['member def modifier' have incorrect indentation level 8, expected level should be 4.](https://www.codacy.com/app/ekho/health-checks/file/2773600378/issues/source?bid=3363458&fileBranchId=3363544#l29)